1.9/administration/metrics/quickstart.md

```diff
@@ -0,0 +1,183 @@
+---
+post_title: Quick Start
+menu_order: 0
+---
+
+Use this guide to get started with the DC/OS metrics component. 
+
+**Prerequisites:** 
+
+- At least one [DC/OS service](/docs/1.9/usage/marathon/application-basics/) is deployed.
+- Optional: the CLI JSON processor [jq](https://github.com/stedolan/jq/wiki/Installation).
+
+The metrics component is natively integrated with DC/OS and no additional setup is required.  
+
+# Gather metrics for all containers running on a host
+
+1.  SSH to the agent node running the service.
+
+    ```
+    $ dcos node ssh --master-proxy --mesos-id=<node-id>
+    ```
+
+1.  Run this command to to show all containers that are deployed on the agent node. 
+
+    ```bash
+    $ curl -s http://localhost:61001/system/v1/metrics/v0/containers | jq
+    ```
+    
+    The output should resemble this:
+    
+    ```bash
+    ["121f82df-b0a0-424c-aa4b-81626fb2e369","87b10e5e-6d2e-499e-ae30-1692980e669a"]
+    ```
+
+# Gather metrics for a container
+    
+1.  SSH to the agent node running the service.
+
+    ```
+    $ dcos node ssh --master-proxy --mesos-id=<node-id>
+    ```
+
+1.  Run this command to view the metrics for a particular container (`<container-id>`):
+
+    ```bash
+    $ curl -s http://localhost:61001/system/v1/metrics/v0/containers/<container-id>/app | jq
+    ```
+    
+    The output should resemble:
+    
+    ```bash
+    {"datapoints":[{"name":"dcos.metrics.module.container_received_bytes_per_sec","value":0,"unit":"","timestamp":"2016-12-13T23:06:26Z"},{"name":"dcos.metrics.module.container_throttled_bytes_per_sec","value":0,"unit":"","timestamp":"2016-12-13T23:06:26Z"}],"dimensions":{"mesos_id":"","container_id":"6972ad7c-1701-4970-ae14-4372f76eda37","executor_id":"confluent-kafka.7aff271b-c182-11e6-a88f-22e5385a5fd7","framework_id":"a29070cd-2583-4c1a-969a-3e07d77ee665-0001","hostname":""}}
+    ```
+    
+# Gather metrics from container-level cgroup allocations
+
+1.  SSH to the agent node running the service.
+
+    ```
+    $ dcos node ssh --master-proxy --mesos-id=<node-id>
+    ```
+
+1.  Run this command to view cgroup allocations for a container. 
+
+    ```bash
+    $ /system/v1/metrics/v0/containers/<container-id> | jq
+    ```
+    
+    The output will contain a `datapoints` array that contains information about container resource allocation and utilization provided by Mesos. For example:
+    
+    ```bash
```

Comment:
I noticed the previous JSON blob wasn't pretty printed. Not sure if we want to avoid doing that to save space here, or if we want to pretty print the previous one. Either way, we probably want to be consistent.